rakelib/pwsh.rake

```diff
@@ -275,6 +276,19 @@ namespace :pwsh do
               ruby_arg:                   'bare',
               validate_not_null_or_empty: true
             }
+          # bolt module show
+          when 'Get'
```

Comment:
Hm, were you able to get this working? It only lists the modules for me with `Get-BoltModule -Name 'puppetlabs-puppet_agent'` (after running both the pwsh cmdlet Rake task and the generate module rake task + importing the module).

Reply:
@lucywyman It's working as expected for me, both with a named argument (`Get-Module -Name puppet_agent`) and a positional argument (`Get-Module puppet_agent`). The `Get-Module` command without any additional args is also still returning the full module list.

```powershell
> get-boltmodule -name puppet_agent

puppetlabs/puppet_agent [4.8.0]
  Upgrades All-In-One Puppet Agents

Tasks
  puppet_agent::delete_local_filebucket     Removes the local filebucket
  puppet_agent::facts_diff                  Run the Puppet agent facts diff action
  puppet_agent::install                     Install the Puppet agent package
  puppet_agent::version                     Get the version of the Puppet agent...

...
```

```powershell
> get-boltmodule puppet_agent

puppetlabs/puppet_agent [4.8.0]
  Upgrades All-In-One Puppet Agents

Tasks
  puppet_agent::delete_local_filebucket     Removes the local filebucket
  puppet_agent::facts_diff                  Run the Puppet agent facts diff action
  puppet_agent::install                     Install the Puppet agent package
  puppet_agent::version                     Get the version of the Puppet agent...

...
```

```powershell
> get-boltmodule

Plan Language Modules
  boltlib   (built-in)
  ctrl      (built-in)
  dir       (built-in)
  file      (built-in)
  log       (built-in)
  out       (built-in)
  prompt    (built-in)
  system    (built-in)

...
```